README.md

```diff
@@ -1,34 +1,42 @@
 # OCaml 5 Tutorial
 
 A hands-on tutorial on the new parallelism features in OCaml 5. This tutorial
-was run on 19th May 2022.
+was run on 5 July 2022. Currently, the alpha version of OCaml 5 has been released, and the full version is set for release in September 2022. 
```

Comment:
We ran the tutorial on 19th May 2022 at the Tarides offsite. I didn't run it on 5th July 2022. 

I wanted to have a date in the README since it is very likely that the tutorial will go out of date with all the breaking changes that are happening as we head towards the release of OCaml 5.

Perhaps we can make it clear by rewording the sentence as "was run on the 19th of May 2022 at the [Tarides retreat](https://tarides.com/blog/2022-06-23-team-tarides-visits-a-17th-century-chateau)".

Reply:
Revised! 
I put that in since I was running it, but it's best to keep it when you ran it fully.